linux_os/guide/system/network/network-nftables/set_nftables_loopback_traffic/rule.yml

```diff
@@ -0,0 +1,43 @@
+documentation_complete: true
+
+title: 'Set nftables configuration for loopback traffic'
+
+description: |-
+    Configure the loopback interface to accept traffic.
+    Configure all other interfaces to deny traffic to the loopback
+    network.
+
+rationale: |-
+    Loopback traffic is generated between processes on machine and is
+    typically critical to operation of the system. The loopback interface
+    is the only place that loopback network traffic should be seen,
+    all other interfaces should ignore traffic on this network as an
+    anti-spoofing measure.
+
+severity: medium
+
+identifiers:
+    cce@sle15: CCE-92481-1
+
+references:
+    cis@sle15: 3.5.2.6 
+    pcidss: Req-1.4.1
+
+warnings:
+    - general: |-
```

Comment:
I also think it would be useful to include a warning informing the remediation is not permanent.